packages/server/customer-os-api/repository/job_role_repository.go

```diff
@@ -92,6 +93,34 @@ func (r *jobRoleRepository) GetAllForContacts(ctx context.Context, tenant string
 	return result.([]*utils.DbNodeAndId), err
 }
 
+func (r *jobRoleRepository) GetAllForUsers(ctx context.Context, tenant string, userIds []string) ([]*utils.DbNodeAndId, error) {
+	span, ctx := opentracing.StartSpanFromContext(ctx, "JobRoleRepository.GetAllForUsers")
+	defer span.Finish()
+	tracing.SetDefaultNeo4jRepositorySpanTags(ctx, span)
+
+	session := utils.NewNeo4jWriteSession(ctx, *r.driver)
+	defer session.Close(ctx)
+
+	result, err := session.ExecuteRead(ctx, func(tx neo4j.ManagedTransaction) (any, error) {
+		if queryResult, err := tx.Run(ctx, `
+			MATCH (:Tenant {name:$tenant})<-[:USER_BELONGS_TO_TENANT]-(u:User)-[:WORKS_AS]->(job:JobRole)
+			WHERE c.id IN $userIds
```

Comment:
correct query; WHERE u.id IN $userIds

Need a test to fetch user with a job role. such test will cover query execution
There is a similar test for contact with job role